site/docs/releases.md

```diff
@@ -67,6 +66,74 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 </dependencies>
 ```
 
+### 1.5.0 release
+
+Apache Iceberg 1.5.0 was released on March 11, 2024.
+The 1.5.0 release adds a variety of new features and bug fixes.
+
+* API
+  - Extend FileIO and add EncryptingFileIO. ([\#9592](https://github.com/apache/iceberg/pull/9592))
+  - Track partition statistics in TableMetadata ([\#8502](https://github.com/apache/iceberg/pull/8502))
+  - Add sqlFor API to views to handle resolving a representation for a dialect([\#9247](https://github.com/apache/iceberg/pull/9247))
+* Core
+  - Add View support for REST catalog ([\#7913](https://github.com/apache/iceberg/pull/7913))
+  - Add view support on the JDBC catalog ([\#9487](https://github.com/apache/iceberg/pull/9487))
+  - Add catalog type for glue,jdbc,nessie ([\#9647](https://github.com/apache/iceberg/pull/9647))
+  - Support Avro file encryption with AES GCM streams ([\#9436](https://github.com/apache/iceberg/pull/9436))
+  - Add ApplyNameMapping for Avro ([\#9347](https://github.com/apache/iceberg/pull/9347))
+  - Add StandardEncryptionManager ([\#9277](https://github.com/apache/iceberg/pull/9277))
+  - Add REST catalog table session cache ([\#8920](https://github.com/apache/iceberg/pull/8920))
+  - Support view metadata compression ([\#8552](https://github.com/apache/iceberg/pull/8552))
+  - Track partition statistics in TableMetadata ([\#8502](https://github.com/apache/iceberg/pull/8502))
+  - Enable column statistics filtering after planning ([\#8803](https://github.com/apache/iceberg/pull/8803))
+* Spark
+  - Remove support for Spark 3.2 ([\#9295](https://github.com/apache/iceberg/pull/9295))
+  - Support creating, dropping and renaming views via SQL ([\#9423](https://github.com/apache/iceberg/pull/9423), [\#9421](https://github.com/apache/iceberg/pull/9421), [\#9343](https://github.com/apache/iceberg/pull/9343))
```

Comment:
```suggestion
  - Support views via SQL for Spark 3.4 and 3.5 ([\#9423](https://github.com/apache/iceberg/pull/9423), [\#9421](https://github.com/apache/iceberg/pull/9421), [\#9343](https://github.com/apache/iceberg/pull/9343))
```

also there were a few more PRs to support views in Spark

Reply:
> there were a few more PRs to support views in Spark

some of them are refactoring and issue fixes. Hence mentioned only the foundations.

Reply:
#9513 and #9582 are the ones that are missing here, since those add remaining functionality

Reply:
done